IBPSA/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo

```diff
@@ -290,7 +290,8 @@ equation
         Rectangle(
           extent={{-80,60},{80,-60}},
           lineColor={0,0,0},
-          fillColor={95,95,95},
+          fillColor=DynamicSelect({95,95,95},
+          min(1, -Q_flow/Q_flow_nominal)*{255,0,0}+(1-min(1, -Q_flow/Q_flow_nominal))*{95,95,95}),
```

Comment:
Not sure what happens when `Q_flow>Q_flow_nominal` and consequently the color > 255. I presume that it simply saturates to the maximum and that this is fine.

Reply:
Good point, I will correct this.